pkgs/racket-doc/scribblings/reference/match.scrbl

```diff
@@ -166,7 +171,12 @@ In more detail, patterns match as follows:
        #:eval match-eval
        (match '(1 2 3 4 5 6)
          [(list-no-order 6 2 y ...) y])
-       ]}
+       ]
+
+       Unlike other patterns, @racketidfont{list-no-order} doesn't
```

Comment:
Instead of putting this comment in 2 places, what about putting it in a margin note in between the relevant bullet points?

Reply:
Done.

Reply:
The margin note looks like this:
<img width="1067" alt="margin-note" src="https://user-images.githubusercontent.com/6600123/38230528-60bcaebe-36dc-11e8-9a82-2a19c6959946.png">

Reply:
Or this with a slightly smaller window size:
<img width="731" alt="margin-note-2" src="https://user-images.githubusercontent.com/6600123/38230615-c1e1a190-36dc-11e8-97e3-86618bf94ac7.png">